app/src/main/java/com/cris/nvh/framgiaproject/MainActivity.java

```diff
@@ -51,14 +57,23 @@
 	protected void onCreate(Bundle savedInstanceState) {
 		super.onCreate(savedInstanceState);
 		setContentView(R.layout.activity_main);
+		initView();
 		initConnection();
-		initHandler();
 		initService();
-		initView();
+		initHandler();
 		initReceiver();
 		registerReceiver();
 	}
 
+	@Override
+	protected void onResume() {
+		super.onResume();
+		if (mService != null) {
+			mService.setUIHandler(mHandler);
```

Comment:
format code
1 tab = 4 space
```suggestion
	    mService.setUIHandler(mHandler);
```